packages/flutter_tools/lib/executable.dart

```diff
@@ -66,6 +67,7 @@ Future<Null> main(List<String> args) async {
     ..addCommand(new RunCommand())
     ..addCommand(new RunMojoCommand(hidden: !verboseHelp))
     ..addCommand(new ScreenshotCommand())
+    ..addCommand(new SkiaCommand())
```

Comment:
Well, I don't know what it does, so I can't really say. But I think we should introduce it to them more in the context of what they're trying to do at the time. So instead of `flutter skia` perhaps `flutter diagnose-drawing` or `flutter diagnose skia`.